### PR TITLE
[review] Use RxJS to make the request

### PR DESCRIPTION
**Pull request by oliviertassinari**
Step after step, I'm setting up all the layers of the stack to display the screenshots 🎉 

Next step: implement the server graphQL resolver for that query. Data will use the seeds to begin with.
Next step: use the seeds data to actually render something with React.

Related to #25

### CODE REVIEW

File: src/review/routes/build/reducer.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+import actionTypes from 'redux/actionTypes';
 
-function reducer(state, action) {
+function buildReducer(state, action) {
   if (state === undefined) {
     state = {};
   }
 
   switch (action.type) {
+    case actionTypes.BUILD_FETCH_TASK:
```
Comment (neoziro) [inline src/review/routes/build/reducer.js]: You should only do this if we are in success isn't it?

File: src/review/routes/build/Build.js [hunk 2]
```diff
@@ -54,4 +45,6 @@ class Build extends Component {
   }
 }
 
-export default connect()(Build);
+export default connect(state => ({
+  state,
```
Comment (neoziro) [inline src/review/routes/build/Build.js]: We don't need to connect all the state here but only the payload. Am I wrong?

Comment (oliviertassinari) [inline src/review/routes/build/Build.js]: You are right, that need some refinement.

File: src/review/routes/build/reducer.js (hunk 1 above)
Comment (oliviertassinari) [inline src/review/routes/build/reducer.js]: I'm planning adding a `watchTask` HoC, so having the progress state is going to help me out.

**Pull request merged**